lib/utils/util.js

```diff
@@ -100,28 +100,16 @@ util.time = function time(date) {
  */
 
 util.hex32 = function hex32(num) {
-  assert((num >>> 0) === num);
+  assert(num >= 0);
```

Comment:
You can use `assert((num >>> 0) === num)` here (make sure num is 32bit) and get rid of `num.length <= 8` assert below.